src/GiftAidField.php

```diff
@@ -65,17 +66,24 @@ public function get_field_input($form, $value = '', $entry = null): string
     {
         $id = (int) $this->id;
         $giftaidImage = plugins_url('public/img/giftaid.svg', __DIR__);
+        $selectedPriceField = $this->selectedPriceField ?? '.ginput_amount';
 
         ob_start();
         ?>
-        <div class="gift-box-wrapper bg-gray-50 rounded-br-4 p-7.5">
+        <div
+            class="gift-box-wrapper bg-gray-50 rounded-br-4 p-7.5"
+            data-selected-price-field-id="<?php echo esc_attr($selectedPriceField); ?>"
```

Comment:
`data-selected-price-field-id` expects a HTML ID, yet `$$selectedPriceField` can be an ID **without** `#` OR it can be a class **with** a `.`.
This is a bug I can see when used in JS